### PR TITLE
chore: Update to latest fabric-peer-ext

### LINKED ISSUE #25: Update with latest fabric-peer-ext
This update changes the config system chaincode to in-process user chaincode.

### PR DESCRIPTION
**Pull request by bstasyszyn**
closes #25

Signed-off-by: Bob Stasyszyn <Bob.Stasyszyn@securekey.com>

### CODE REVIEW

File: cmd/ledgerconfig/deletecmd/deletecmd.go [hunk 1]
```diff
@@ -127,7 +127,7 @@ func (c *command) run() error {
 		return err
 	}
 
-	_, err = ch.Execute(req, channel.WithTargetEndpoints(c.Context().Peers...))
+	_, err = ch.Execute(req)
```
Comment (troyronda) [inline cmd/ledgerconfig/deletecmd/deletecmd.go]: The option isn't needed anymore?

File: test/bddtests/features/ledgerconfig.feature [hunk 2]
```diff
@@ -10,6 +10,7 @@ Feature: ledger-config
 
   Background: Setup
     Given the channel "mychannel" is created and all peers have joined
+    And "test" chaincode "configscc" is instantiated from path "in-process" on the "mychannel" channel with args "" with endorsement policy "AND('Org1MSP.member','Org2MSP.member')" with collection policy ""
```
Comment (Baha-sk) [inline test/bddtests/features/ledgerconfig.feature]: if I recall the previous PR, it's called `configcc` now (ie not `confiscc`), does this apply here?

File: cmd/ledgerconfig/deletecmd/deletecmd.go (hunk 1 above)
Comment (bstasyszyn) [reply cmd/ledgerconfig/deletecmd/deletecmd.go]: No. The config chaincode is now a user chaincode that specifies a policy. So, the targets should be resolved by the Discovery service according to the policy.

File: test/bddtests/features/ledgerconfig.feature (hunk 2 above)
Comment (bstasyszyn) [reply test/bddtests/features/ledgerconfig.feature]: I think it has always been configscc. Maybe configcc is more appropriate. Possibly in a future change.

Comment (Baha-sk) [reply test/bddtests/features/ledgerconfig.feature]: my bad, that's in a different project.. never mind my previous comment then.

**Pull request merged**